Classes/Manager/FLEXManager+Extensibility.h

```diff
@@ -25,7 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 /// The objectFutureBlock will be invoked from the main thread and may return nil.
 /// @note The passed block will be copied and retain for the duration of the application,
 /// you may want to use __weak references.
-- (void)registerGlobalEntryWithName:(NSString *)entryName objectFutureBlock:(id (^)(void))objectFutureBlock;
+- (void)registerGlobalEntryWithName:(NSString *)entryName objectFutureBlock:(id (^)(void))objectFutureBlock __attribute((deprecated("Use the same method on the globalEntriesContainer property instead.")));
```

Comment:
I see no reason to depreciate the old method, I'm not in a hurry to add more warnings for people to deal with
```suggestion
- (void)registerGlobalEntryWithName:(NSString *)entryName objectFutureBlock:(id (^)(void))objectFutureBlock;
```